integration/clean/README.md

```diff
@@ -0,0 +1,3 @@
+**What does the cleaner do?**
+
+Cleaner cleans out old ami (ami older than 60 days)
```

Comment:
Not sure what @jhnlsn was looking for, but for me, I would like to see more of an explanation here as to what the cleaner actually does (I know that it cleans out old AMIs), and why we need to have this.